### PR TITLE
Add link to the code of conduct from contributing guides.

### PR DESCRIPTION
**Pull request by jacobian**
This is inspired by Angluar.js
(https://github.com/angular/angular/blob/master/CONTRIBUTING.md), among others.
Thanks to Jeff Triplett for the prod.

### CODE REVIEW

**Comment by jefftriplett** [comment]
Thank you for the quick response!

File: docs/intro/contributing.txt [hunk 1]
```diff
@@ -68,6 +68,12 @@ It contains lots of great information and is a must read for anyone who'd like
 to become a regular contributor to Django. If you've got questions, it's
 probably got the answers.
 
+Conduct
```
Comment (MarkusH) [inline docs/intro/contributing.txt]: I'd use "Code of Conduct" in the headline too.

**Comment by carljm** [comment]
:+1:

**Comment by timgraham** [comment]
merged in f043434174db3432eb63c341573c1ea89ef59b91, thanks!

**Pull request closed without merging**